lib/tasks/reports.rake

```diff
@@ -18,7 +18,7 @@ namespace :reports do
     conditions[:timerange] = ENV['days'].to_i.days if ENV['days']
     conditions[:status] = ENV['status'].to_i if ENV['status']
 
-    Report.expire(conditions)
+    ConfigReport.expire(conditions)
```

Comment:
Perhaps make this an argument so the same task can be re-used for different subclasses.